### ISSUE TITLE
Service definition `valetudo.conf` went missing?

### ISSUE THREAD

**Issue by jnj16180340** [Hypfer/Valetudo#452]
## Describe the bug
TL;DR `/etc/init/valetudo.conf` disappeared; manually copying `/deployment/valetudo.conf` restores functionality.          

All other Valetudo files are untouched -- At least, it seems to work fine and config settings / zones / etc are unchanged. When the service def was missing, Valetudo still ran fine launched manually through ssh        

Disk space looks fine, etc. I'll update this if it happens again.

## To Reproduce
Unknown -- Happened about 4 months after installation, maybe 20-30 vacuum runs??

### Vacuum Model
- Untouched except through web interface since flashing fw+valetudo a few months ago
- Gen2
- FW version: 3.3.9
- FW build: 003468
- Name: A.03.0023_PRC
- Bom: A.03.0023
- Location: prc
- Language: prc
- Logserver: awsbj0.fds.api.xiaomi.com

### Valetudo Version
0.4.0

**Comment by rumpeltux** [comment]
Thanks for the report. Due to missing logs and repro instructions, I don’t think there’s anything we can do at this point. Feel free to reopen if this changes.
